kola/tests/ignition/filesystem.go

```diff
@@ -253,6 +257,53 @@ func init() {
 		UserData:    swapConfigV2_1,
 		Distros:     []string{"cl"},
 	})
+
+	swapActivation := conf.ContainerLinuxConfig(fmt.Sprintf(`storage:
+  files:
+  - path: /etc/sysctl.d/80-swappiness.conf
+    filesystem: root
+    mode: 0644
+    contents:
+      inline: "vm.swappiness=10"
+
+systemd:
+  units:
+    - name: var-vm-swapfile1.swap
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Turn on swap
+        Requires=create-swapfile.service
+        After=create-swapfile.service
+
+        [Swap]
+        What=%[1]s
+
+        [Install]
+        WantedBy=multi-user.target
+    - name: create-swapfile.service
+      contents: |
+        [Unit]
+        Description=Create a swapfile
+        RequiresMountsFor=/var
+        ConditionPathExists=!%[1]s
```

Comment:
This makes this required unit fail to start which, e.g., could happen on reboots, I would move this check into `ExecStart`

Reply:
(or just unconditionally run `fallocate` and `mkswap` again if it's idempotent and will succeed)

Reply:
`fallocate` and `mkswap` look idempotent - will remove the condition:
```
$ fallocate -l 1024m ./test
$ echo $?
0
$ chmod 600 ./test
$ mkswap ./test
Setting up swapspace version 1, size = 1024 MiB (1073737728 bytes)
no label, UUID=1b298276-fde1-4320-b4ee-092e04548db6
$ fallocate -l 1024m ./test
$ echo $?
0
$ mkswap ./test
mkswap: ./test: warning: wiping old swap signature.
Setting up swapspace version 1, size = 1024 MiB (1073737728 bytes)
no label, UUID=9e988d79-78cb-4e4e-9ee0-9535bc1a59f5
$ echo $?
0
```